package.json

```diff
@@ -6,8 +6,7 @@
   "scripts": {
     "check": "prettier --check .",
     "fix": "prettier --write .",
-    "start": "node server.js",
-    "dev": "nodemon server.js"
+    "start": "node server.js"
```

Comment:
Why are we removing nodemon?

Reply:
Any changes made in any file will restart the server but does not refresh the page

Reply:
makes sense